providers/fips/selftest.c

```diff
@@ -50,6 +50,10 @@ DEFINE_RUN_ONCE_STATIC(do_fips_self_test_init)
     return self_test_lock != NULL;
 }
 
+#define DEP_DELCARE()                                                          \
```

Comment:
Who's care? 😉

Reply:
Not me :) Least I spelt it consistently wrong..
Updated